test/io/ferguson/digits.execopts

```diff
@@ -1,4 +1,4 @@
 --test=1
 --test=2
---test=3
+--test=3 # digits.binary-string.good
```

Comment:
That looks like that specifies a .good file, but I don't see one added to this PR.  Was it already present, or did you forget to add the new .good file?